scripts/post-install.js

```diff
@@ -46,6 +46,13 @@
 
 const chalk = require('chalk');
 
+// SKIP_NRFJPROG_CHECK environment variable allows to skip checking
+// for the purpose of publishing job on build server
+if (process.env.SKIP_NRFJPROG_CHECK) {
```

Comment:
I see that this only checks for the presence of `SKIP_NRFJPROG_CHECK`. So, I guess `SKIP_NRFJPROG_CHECK=false` would skip the check, since "false" is a string? Maybe only skip if the value is "true"?